### ISSUE TITLE
Build error "Undefined symbol: _OBJC_CLASS_$_CTTelephonyNetworkInfo" on Catalyst

### ISSUE THREAD

**Issue by megabitsenmzq** [microsoft/appcenter-sdk-apple#2359]
### **Description**

Build error "Undefined symbol: _OBJC_CLASS_$_CTTelephonyNetworkInfo" on Catalyst

Undefined symbols for architecture x86_64:
  "_OBJC_CLASS_$_CTTelephonyNetworkInfo", referenced from:
      objc-class-ref in AppCenter.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)

### **Repro Steps**

Please list the steps used to reproduce your issue.

1. Add AppCenter to project use SwiftPM
2. Enable Catalyst
3. Build

### **Details**

1. Which SDK version are you using?
    4.3.0
2. Which OS version did you experience the issue on?
    macOS 11.6 (Catalyst)
3. Which Xcode version did you build the app with?
    Xcode 13.0 (13A233)
5. Which Cocoapods version are you using (run `pod --version`)?
   Not using cocoapods
6. What device version did you see this error on?  Were you using an emulator or a physical device?
    MacBook Pro (Retina, 15-inch, Mid 2015)
7. What language are you using?
    Swift
8. What third party libraries are you using?
9. Please enable verbose logging for your app using `MSAppCenter.setLogLevel(.verbose)` before your call to `MSAppCenter.start(...)` for Swift, or `[MSAppCenter setLogLevel:MSLogLevelVerbose]` before `[MSAppCenter start: ...]` for Objective C and include the logs here:

**Comment by DmitriyKirakosyan** [comment]
Hi @megabitsenmzq 

Thank you for bringing this issue to our attention!

I was able to reproduce the issue and created a bug on our board.
We will let you know once the fix is released.

Meanwhile, as a workaround, please add CoreTelephony.framework to your project's target.

**Comment by megabitsenmzq** [comment]
Thank you so much! 🥳

**Comment by CristianMoisei** [comment]
Thanks @DmitriyKirakosyan for updating this, looking forward to the release.

**Comment by DmitriyKirakosyan** [comment]
Hi guys, the fix was merged into `develop` branch, so you can use it now to fix the issue.
I'll update this thread once the fix is merged to master during the next release.

**Comment by aleksandr-dorofeev** [comment]
We've included fixes in the latest SDK release for this. I'm closing the issue but please reopen if you run into this again.
